next/components/forms/segments/AccountSections/MyApplicationsSection/MyApplicationDetails.tsx

```diff
@@ -0,0 +1,78 @@
+import MyApplicationDetailsHeader from 'components/forms/segments/AccountSections/MyApplicationsSection/MyApplicationDetailsHeader'
+import MyApplicationHistory, {
+  MyApplicationHistoryDataBase,
+} from 'components/forms/segments/AccountSections/MyApplicationsSection/MyApplicationHistory'
+import { MyApplicationsSentCardBase } from 'components/forms/segments/AccountSections/MyApplicationsSection/MyApplicationsSentList'
+import SummaryRow from 'components/forms/steps/Summary/SummaryRow'
+import { useTranslation } from 'next-i18next'
+
+type MyApplicationsDetailsBase = {
+  detailsData: MyApplicationsSentCardBase
+  historyData: MyApplicationHistoryDataBase[]
+}
+
+const MyApplicationDetails = ({ detailsData, historyData }: MyApplicationsDetailsBase) => {
+  const { t } = useTranslation('account')
+  return (
+    <div className="flex flex-col">
+      <MyApplicationDetailsHeader data={detailsData} />
+      <div className="w-full max-w-screen-lg mx-auto flex flex-col py-12 gap-16">
+        <div className="flex flex-col gap-2 px-4 lg:px-0">
+          <h3 className="text-h3">
+            {t('account_section_applications.details.application_details.title')}
+          </h3>
+          <div className="w-full flex flex-col">
+            <SummaryRow
+              size="small"
+              isEditable={false}
+              data={{
+                label: t('account_section_applications.details.application_details.record_number'),
+                value: 'MAG 12345/2023',
```

Comment:
would prefer for these values to come from mockups the same way as the application details do